frontend/apps/marketing/src/components/link/link.module.scss

```diff
@@ -0,0 +1,53 @@
+a.link {
+  margin-top: 0;
+}
+
+// Currently purple color styles are match component-library styles completely so we are not editing it.
+//a.link-purple {
+//}
```

Comment:
Do we need this here at all?

Reply:
It's a nice to have for the future, not a need to have.
I added this to eliminate possible questions re why we don't have .link-purple styles while passing it as a className to the prop by documenting it in this way.
Plus, in case we will need to extend `link-purple` styles in the future - all we will need to do is just uncomment this and start writing the actual styles, without worrying that these styles will be in the correct place

Reply:
Small addition for context: if it was in `packages/component-library` - we would not have this problem since we control/adjust everything there, meanwhile in `apps/marketing` we rather make adjustments to existing styles and/or adding new custom styles, meaning some degree of documenting might be really helpful for the people of future.

Good part here is that I think once we agree on something here, we may follow it for possible future cases

Reply:
Ok, I removed this comment since it raises more questions then answers and looks like color logic will be refactored pretty soon anyways.

> Like if marketing wants to add a pink button we can do that here? It might help to add a comment explaining that like Leaving this here in case we want to add additional color options to the marketing sites.

Not exactly. It was there just to show that .link-purple className actually exists and it's empty because Currently purple color styles are match component-library styles completely so we are not editing it. (Specifically what was commented there)
In order to add more colors support - just adding new className won't be enough, since there're much more then this one step. (definitions,props,etc)